example/plugin-function-tests/Cargo.toml

```diff
@@ -0,0 +1,21 @@
+[package]
+name = "plugin-function-tests"
```

Comment:
If it's a test, it should not in example; if it's in example, it should has "test" in name

Reply:
Sure thing, updated.